docs/release_notes.rst

```diff
@@ -4,6 +4,13 @@ Release Notes
 **pydocstyle** version numbers follow the
 `Semantic Versioning <http://semver.org/>`_ specification.
 
+Current Development Version
+---------------------------
+
+Bug Fixes
+
+* Fix bug with google style arg regex (#448).
```

Comment:
Capitalize Google, and make this more descriptive as to what was the original issue, e.g., "Fix a bug in parsing Google-style argument description. The bug caused some argument names to go unreported in D417 (#448)."